### PR TITLE
Branch about us

### LINKED ISSUE #35: Update AboutUs.md
Add details of developers of Nuudle

### PR DESCRIPTION
**Pull request by YangYue128-helen**
Update AboutUs

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -38,15 +38,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
-### Jean Doe
+### Yang Yue
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/yangyue128-helen.png" width="200px">
 
-[[github](http://github.com/johndoe)]
+[[github](https://github.com/YangYue128-helen)]
 [[portfolio](team/johndoe.md)]
```
Comment (Avalionnet) [inline docs/AboutUs.md]: Hmm maybe you can consider updating this too as they will probably ask us to do it in the weeks ahead. But it's not essential for this iteration so no worries!

**Review by Avalionnet — APPROVED** [review]
LGTM!! Do check out the comment too!

**Review by JinHao-L — APPROVED** [review]
LGTM! 👍

**Review by WadeForward — COMMENTED** [review]
LGTM! 👍

**Pull request merged**